internal/model/docker_info.go

```diff
@@ -188,4 +215,13 @@ type FastBuild struct {
 
 func (FastBuild) buildDetails() {}
 
+type CustomBuild struct {
+	Command string
+	Deps    []string
```

Comment:
I was confused by this, could we use a comment or rename or a type other than `string` - `DependentTargets`? `FileDeps`? `[]LocalPath`?